lab-express-cinema/routes/index.js

```diff
@@ -0,0 +1,53 @@
+const express = require('express');
+const Movie = require("../models/movie.js");
+const router  = express.Router();
+
+/* GET home page */
+router.get('/', (req, res, next) => {
+  res.render('index');
+});
+
+// router.get('/movies',(req,res,next)=>{
+//         res.render('movies.hbs')
+//     })
+
+router.get('/movies',(req,res,next)=>{
+  Movie.find()
+    .then(movies => {
+      res.render('movies.hbs')
+    })
+    .catch((err)=> {
+      next (err);
+    });
+});
+
+// // To link here it's "/read?b=9999"
+// router.get("/watch",(req,res,next)=>{
+//   // const b = req.query.b
+//   const{m}=req.query;
+//   // Book.findOne({_id : b})
+//   Movie.findById(m)
+//   .then((movieDoc)=>{
+//     res.locals.movieItem = movieDoc;
+//     res.render("book-details.hbs")
+//   })
+//   .catch((err)=>{
+//     // show error pages
+//     next(err);
+//   })
+// })
+
+router.get("/movies", (req, res, next) => {
```

Comment:
Be careful, your website wasn't displaying anything because you have a conflict between two routes, they are both listening to the same request get("/movies), but as the first one is only rendering an .hbs file without any data, you website was a bit empty ;)